scripts/sct_propseg.py

```diff
@@ -15,6 +15,58 @@
 import sys
 import sct_utils as sct
 import os
+from scipy import ndimage as ndi
+import numpy as np
+from sct_image import orientation
+
+
+def check_and_correct(fname_segmentation, fname_centerline, threshold_distance=5.0, verbose=0):
+    # convert segmentation image to RPI
+    im_input = Image(fname_segmentation)
+    image_input_orientation = orientation(im_input, get=True, verbose=False)
+    sct.run('sct_image -i ' + fname_segmentation + ' -setorient RPI -o tmp.segmentation_RPI.nii.gz', verbose)
+    sct.run('sct_image -i ' + fname_centerline + ' -setorient RPI -o tmp.centerline_RPI.nii.gz', verbose)
+
+    # go through segmentation image, and compare with centerline from propseg
+    im_seg = Image('tmp.segmentation_RPI.nii.gz')
+    im_centerline = Image('tmp.centerline_RPI.nii.gz')
+
+    # Get size of data
+    sct.printv('\nGet data dimensions...', verbose)
+    nx, ny, nz, nt, px, py, pz, pt = im_seg.dim
+
+    centerline, key_centerline = {}, []
+    for i in range(nz):
+        slice = im_centerline.data[:, :, i]
+        if np.any(slice):
+            x_centerline, y_centerline = ndi.measurements.center_of_mass(slice)
+            centerline[str(i)] = [x_centerline, y_centerline]
+            key_centerline.append(i)
+
+    for i in range(nz):
+        slice = im_seg.data[:, :, i]
+        to_remove = False
+        label_objects, nb_labels = ndi.label(slice)
+        if nb_labels > 1:
```

Comment:
I would add a comment like "check if there is more than one closed object within the slice"